
<launch>

    <arg name="raw_folder" default="complete_model"/>
    
    <include file="$(find trajectory_visualizer)/launch/trajectory_visualizer.launch"/> 
    <include file="$(find aruco_ros)/launch/marker_publisher.launch">
        <arg name="ref_frame" value="base_footprint"/>
    </include>

    <include file="$(find trajectory_refinement)/launch/gripper_wrt_ee_publisher.launch"/> 

    <node pkg="learning_from_demonstration" name="learning_from_demonstration" 
    type="lfd_node.py" output="screen">

    <param name="raw_folder" value="$(arg raw_folder)"/>

    </node>

</launch>